ports/secretteagarden/port.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": 2,
+    "name": "secretteagarden.zip",
+    "items": [
+        "Secret Tea Garden.sh",
+        "secretteagarden"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Secret Tea Garden",
+        "porter": [
+            "MadShmupper"
+        ],
+        "desc": "Make cups of tea for the thirsty townspeople within a time limit, all while dodging dangerous hazards. ",
+        "inst": "Download- [Angus Beer] (https//:angusbeer.itch.io/secret-tea-garden)/Install- Extract and place data.win inside gamedata folder",
+        "genres": [
+            "puzzle",    
+        ],
+        "image": null,
+        "rtr": false,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
+        "source": "Unknown",
+        "md5": null,
+        "status": "Installed"
+    },
+    "files": {
+        "port.json": "secretteagarden/port.json",
+        "Secret Tea Garden.sh": "Secret Tea Garden.sh",
+        "secretteagarden": "secretteagarden"
+    }
```

Comment:
This port.json is from an installed setup - this section isn't supposed to exist on a fresh deploy:
```suggestion
```
It is recommended to create a fresh port.json if necessary using the port generator page.